apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -41,6 +42,67 @@ export class CommandLineConfiguration {
     return new CommandLineConfiguration(commandLineJson);
   }
 
+  /**
+   * Loads the configuration from the specified file and applies any ommited default build
+   * settings.  If the file does not exist, then an empty default instance is returned.
+   * If the file contains errors, then an exception is thrown.
+   */
+  public static loadFromFileAndDefault(jsonFilename: string): CommandLineConfiguration {
```

Comment:
Wouldn't loadFromFile*Or*Default make more sense?

Reply:
Yes it would, I'll update this.